GTG/core/timer.py

```diff
@@ -70,15 +75,55 @@ def time_changed(self):
         self.timeout_source = GObject.timeout_add_seconds(secs_to_refresh,
                                                           self.emit_refresh)
 
-    def set_configuration(self, refresh_hour, refresh_min):
+    def set_configuration(self, time):
         try:
-            datetime.time(int(refresh_hour), int(refresh_min))
+            time
         except (ValueError, TypeError):
-            raise ValueError("{}:{} is not a valid time".format(refresh_hour,
-                                                                refresh_min))
+            raise ValueError("{} is not a valid time".format(time))
```

Comment:
so it's removed and I kept there only the two lines below: setting 'hour' & 'min'